samples/client/petstore/go-experimental/go-petstore/api/openapi.yaml

```diff
@@ -2090,10 +2093,12 @@ components:
       properties:
         breed:
           type: string
+      type: object
```

Comment:
Here I agree `type: object` was missing. It it there in the original document (modules/openapi-generator/src/test/resources/2_0/petstore-with-fake-endpoints-models-for-testing.yaml), it makes sense to add it.